spec.html

```diff
@@ -8567,6 +8567,13 @@ <h1>[[Get]] ( _P_, _Receiver_ )</h1>
           1. Assert: _binding_ is a ResolvedBinding Record.
           1. Let _targetModule_ be _binding_.[[Module]].
           1. Assert: _targetModule_ is not *undefined*.
+          1. If _binding_.[[BindingName]] is `"*namespace*"`, then
+            1. Return ? GetModuleNamespace(_targetModule_).
+          1. Else,
```

Comment:
Please remove the complete `Else` branch, because it's redundant with the following steps.